Sources/Testing/Expectations/Expectation.swift

```diff
@@ -27,7 +26,6 @@ public struct Expectation: Sendable {
   /// If this expectation passed, the value of this property is `nil` because
   /// the difference is only computed when necessary to assist with diagnosing
   /// test failures.
-  @_spi(ForToolsIntegrationOnly)
```

Comment:
This property should probably stay SPI for now as it's a bit hacky and probably not the final design for this data.